web/themes/custom/novel/templates/components/breadcrumb.html.twig

```diff
@@ -1,15 +1,5 @@
-<div class="p-8">
-  <nav role="navigation">
-    <h2 class="hide-visually">
-      {{ 'Breadcrumb'|t }}
```

Comment:
This is removed on purpose - as we use a `<nav>`, it will be plenty for the screenreaders.